go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
 	github.com/jbenet/go-temp-err-catcher v0.1.0
 	github.com/jbenet/goprocess v0.1.4
-	github.com/libp2p/go-doh-resolver v0.3.1
+	github.com/libp2p/go-doh-resolver v0.0.0-20211210150402-67a10ef63bf3
```

Comment:
this needs tagged version.

Reply:
it does. The tagged version has not been released yet, therefore the mention to the commit directly, and a reference to this update in the PR message.
Release is tracked in https://github.com/libp2p/go-doh-resolver/issues/15

Reply:
yeah, just putting an anchor to replace with release tag before merging.

Reply:
The change is now included in `libp2p/go-doh-resolver@0.4.0`.
go-ipfs-config update is likely to be in `ipfs/go-ipfs-config@0.19.0`.

@aschmahmann do you know if there are other changes to be made to go-ipfs-config for go-ipfs 0.13 release? If not, I can update this commit to have the correct version and hash in `go.mod` and `go.sum`.